### ISSUE TITLE
Carthage support

### ISSUE THREAD

**Issue by philipheinser** [ParsePlatform/ParseFacebookUtils-iOS#21]
Please add carthage support to this project.

**Comment by AndrewSB** [comment]
@nlutsenko now that https://github.com/ParsePlatform/Parse-SDK-iOS-OSX is integrated (or close to) it would be awesome to see all the other frameworks given the same treatment :smile: 

Tried to contribute a little but I'm not completely sure what's wrong. Right now a carthage update pointed to the head of this repo gives me

```
`*** Cloning ParseFacebookUtils-iOS
*** Checking out ParseFacebookUtils-iOS at "d4e53b70d9debc8c7e92cedbf2fcaf8f7de4e34c"
*** xcodebuild output can be found in /var/folders/j0/1rklds4n5kz3hl63gj024km40000gn/T/carthage-xcodebuild.W1xh8B.log
*** Skipped building ParseFacebookUtils-iOS due to the error:
Dependency "ParseFacebookUtils-iOS" has no shared framework schemes for any of the platforms: iOS

If you believe this to be an error, please file an issue with the maintainers at https://github.com/ParsePlatform/ParseFacebookUtils-iOS/issues/new
```

**Comment by kyledecot** [comment]
+1

**Comment by nlutsenko** [comment]
We are currently blocked by FBSDK adding support for Carthage, since we need all dependencies to support it before we can support it in this repo.
I'll keep this issue updated on the progress there.

**Comment by AndrewSB** [comment]
@nlutsenko noticed that https://github.com/facebook/facebook-ios-sdk didn't have any issues on the repo, so is Carthage support for FBSDK being internally at Facebook?

**Comment by nlutsenko** [comment]
@AndrewSB, yup, since Issues on that repo is closed - all the communication and tracking on that piece happens internally.

**Comment by AndrewSB** [comment]
@nlutsenko any updates with FBSDK support?

**Comment by AndrewSB** [comment]
@nlutsenko any updates with FBSDK support?

**Comment by teklabs** [comment]
Is it possible to fork some of the repos, add Carthage support on our own until FB adds support?

**Comment by austiine04** [comment]
any luck with this issue? 
did it get fixed?

**Comment by RushRunnerBoston** [comment]
+1

**Comment by antigp** [comment]
#71
